core/ui/src/main/kotlin/com/google/samples/apps/nowinandroid/core/ui/NewsFeed.kt

```diff
@@ -101,7 +103,11 @@ fun launchCustomChromeTab(context: Context, uri: Uri, @ColorInt toolbarColor: In
         .setDefaultColorSchemeParams(customTabBarColor)
         .build()
 
-    customTabsIntent.launchUrl(context, uri)
+    try {
+        customTabsIntent.launchUrl(context, uri)
+    } catch (e: ActivityNotFoundException) {
+        Toast.makeText(context, R.string.custom_tab_activity_not_found, Toast.LENGTH_LONG).show()
```

Comment:
Ideally we wouldn't use a `Toast` to display error messages as it's not great for accessibility and we don't use them anywhere else, however, since this is very much an edge case (I don't think it's that common for users to _not_ have a browser installed but feel free to correct me if I'm missing something obvious) and we already have a `Context` I think it's fine. 

A more scalable solution for handling errors could be something like: pass in an `onLaunchFailed` lambda which then exposes the error via AppState and the app displays it in a SnackBar (similar to how loss of network connectivity is handled).

Reply:
Agree. Also, having to deal with multiple "errors" states makes everything more complicated to handle. Which one should have the priority, are they "stack-able", dismissible, actionable, etc.
I personally feel like `Toast`s are a good enough solution for these kind of transient states that are very uncommon.